test/functional/mempool_updatefromblock.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test mempool descendants/ancestors information update.
+
+Test mempool update of transaction descendants/ancestors information (count, size)
+when transactions have been re-added from a disconnected block to the mempool.
```

Comment:
Maybe hints directly to `UpdateTransactionsFromBlock`

Reply:
> Maybe hints directly to `UpdateTransactionsFromBlock`

This could be more appropriate for unit tests rather for functional ones, no?

Reply:
I'm don't have a strong opinon, but sometimes for newcomers is great to lead them directly to what part of the codebase is tested.